docs/topics/proc_deploying-the-circuit-breaker-booster-using-the-oc-cli-client.adoc

```diff
@@ -66,7 +67,7 @@ These commands use the xref:about-nodeshift[Nodeshift] `npm` module to install y
 endif::built-for-nodejs[]
 
 
-. Check the status of your booster and ensure your pod is running.
+. Check the status of your example and ensure your pod is running.
```

Comment:
Check the status of your example and--> Check the status of your example application and